molgenis-data/src/main/java/org/molgenis/data/Repository.java

```diff
@@ -45,9 +45,9 @@
 	Entity findOne(Object id);
 
 	/**
-	 * find entities based on a stream of ids.
+	 * find entities based on a stream of ids
 	 * 
-	 * @return (empty) Iterable where the order of entities matches the order of ids, never null
+	 * @return (empty) Iterable, never null
```

Comment:
How so never null? What's supposed to happen if one of the rows isn't found in the repository?